images/router/haproxy/Dockerfile

```diff
@@ -8,7 +8,10 @@ FROM openshift/origin-haproxy-router-base
 ADD conf/ /var/lib/haproxy/conf/
 ADD reload-haproxy /var/lib/haproxy/reload-haproxy
 ADD bin/openshift /usr/bin/openshift
-RUN ln -s /usr/bin/openshift /usr/bin/openshift-router
+RUN ln -s /usr/bin/openshift /usr/bin/openshift-router && \
```

Comment:
@bparees / @adellape ^ (assuming that this is actually the thing that we have to do in the event Eric has other options)